.github/workflows/build-images.yaml

```diff
@@ -30,26 +30,39 @@ on:
         description: 'Should images be uploaded to neon ECR'
         type: boolean
         required: false
+      arch:
+        description: 'Architecture to build for'
+        type: string
+        required: false
+        default: 'amd64'
+      matrix-step-name:
+        required: false
+        type: string
+      matrix-key:
+        required: false
+        type: string
```

Comment:
I see it's used in a workaround later, could you please add some more information about it? (maybe we can do it without that)